client/signup/steps/plans/index.jsx

```diff
@@ -323,6 +348,9 @@ export default connect(
 		isLoadingExperiment: isLoading( state ),
```

Comment:
Is this pre-existing experiment loading state covering your usage?

Reply:
Yes, this covers my usage.